setup.py

```diff
@@ -44,19 +70,30 @@ def get_version(module='spyder_kernels'):
 setup(
     name='spyder-kernels',
     version=get_version(),
-    keywords='spyder jupyter kernel',
+    keywords='spyder jupyter kernel ipython console shell',
     url='https://github.com/spyder-ide/spyder-kernels',
+    download_url="https://www.spyder-ide.org/#fh5co-download",
     license='MIT',
     author='Spyder Development Team',
-    description='Jupyter kernels for the Spyder console',
+    author_email="spyderlib@googlegroups.com",
+    description="Jupyter kernels for the Spyder IDE's IPython consoles.",
+    long_description="\n".join(DOCLINES[4:]),
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=REQUIREMENTS,
     include_package_data=True,
+    platforms=["any"],
```

Comment:
No, that's not true. Please remove this.

Reply:
Okay, removed. The reason I added it was because ``'Operating System :: OS Independent',`` was included as a trove classifier. Should this, then, also be removed, or does ``spyder-kernels`` fall into some edge case where one is true but not the other? (Also, as a sidenote, Spyder itself was marked ``"any"`` in its setup.py until my recent PR fixed it).